tmt/steps/discover/fmf.py

```diff
@@ -430,30 +442,30 @@ def assert_git_url(plan_name: Optional[str] = None) -> None:
                             shutil.copytree(
                                 git_root, self.testdir, symlinks=True)
 
-            # Initialize or remove fmf root
-            if dist_git_init:
-                if not dist_git_extract:
-                    dist_git_extract = sourcedir
-                if not self.opt('dry'):
-                    fmf.Tree.init(dist_git_extract)
-            elif dist_git_remove_fmf_root:
-                if not self.opt('dry'):
-                    shutil.rmtree(os.path.join(
-                        dist_git_extract or top_fmf_root, '.fmf'))
-                if not dist_git_extract:
-                    dist_git_extract = sourcedir
-
-            # Now can safely default to top_fmf_root
+            # Set dist_git_extract if not set yet
             if not dist_git_extract:
-                dist_git_extract = top_fmf_root
+                dist_git_extract = [{
+                    "src": sourcedir,
+                    "dest": '.',
+                }]
 
             # Now copy dist_git_extract into tests
             if not self.opt('dry'):
-                tmt.utils.copytree(
-                    dist_git_extract,
-                    self.testdir,
-                    symlinks=True,
-                    dirs_exist_ok=True)
+                # Copy all requested paths
+                for path in dist_git_extract:
```

Comment:
But `path` is not actually a path, is it? It's a mapping, src/dst pair - but with my proposal of changing this to `Path, Path` tuples, we could do a nice `for src, dst in actual_dist_git_extract` expansion :)

Reply:
Hmm indeed should use a different naming, but it should be consistent with the spec file format for simplicity